src/tests/super-admins.test.js

```diff
@@ -1,5 +1,58 @@
-// Basic test to avoid conflicts
-// REMOVE THIS TEST AND MAKE YOURS!
-test('should pass a basic test', () => {
-  expect(1 + 1).toBe(2);
+import request from 'supertest';
```

Comment:
Missing status 500 test for all endpoints

Reply:
ready.